### PR TITLE
add README file with CRUD implementation

### PR DESCRIPTION
**Pull request by ValeriiaTen**
1. automatic change of the structure of the code using eslint
2. create the table in README file with CRUD

### CODE REVIEW

File: checkIn/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# CRUD Implementation
```
Comment (rvrita) [inline checkIn/README.md:1]: 🔥

File: checkIn/client/public/bundle.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ eval("var api = __webpack_require__(/*! ../../../node_modules/style-loader/dist/
 /***/ (function(module, __webpack_exports__, __webpack_require__) {
```
Comment (rvrita) [inline checkIn/client/public/bundle.js:105]: I think you don't need the bundle here :)

File: checkIn/server/db.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const {
   BOOLEAN,
 } = require('sequelize');
 
-const password = 'secret_Wilson10!';
+const password = 'Richi53!';
```
Comment (rvrita) [inline checkIn/server/db.js:9]: I don't know sequelize very much but committing a variable called password does not feel right, can you double check?

**Review by rvrita — APPROVED** [review]
I just had one comment about the password otherwise it looks good to me!

File: checkIn/server/db.js (hunk 3 above)
Comment (ValeriiaTen) [reply checkIn/server/db.js:9]: Good catch! Thank you

**Pull request merged**